src/assets/icons/dollar.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" width="24" height="24" stroke-width="1">
+  <path d="M12.696093750000001 18.0v-1.0617187499999998c1.0851562499999998-0.056249999999999994 1.91015625-0.3515625 2.54765625-0.8859374999999999 0.6375-0.534375 0.9562499999999999-1.24453125 0.9562499999999999-2.0671875 0.0-0.8859374999999999-0.2578125-1.5398437500000002-0.8273437499999999-1.9546875000000001-0.5765625000000001-0.47109375000000003-1.4015624999999998-0.8296874999999999-2.6156249999999996-1.0617187499999998h-0.060937500000000006V8.2453125c0.8273437499999999 0.11953124999999999 1.5304687499999998 0.3515625 2.2289062499999996 0.8296874999999999l1.021875-1.30078125c-1.021875-0.6539062499999999-2.1070312500000004-1.00546875-3.2531250000000003-1.0617187499999998V6.0h-0.9562499999999999v0.71015625c-0.9562499999999999 0.056249999999999994-1.7882812499999998 0.3515625-2.42578125 0.8859374999999999-0.6375 0.534375-0.9562499999999999 1.24453125-0.9562499999999999 2.0671875s0.2578125 1.4765625 0.8273437499999999 1.8914062500000002c0.50859375 0.41484374999999996 1.4015624999999998 0.7664062500000001 2.54765625 1.00546875v2.77734375c-0.8953125000000001-0.11953124999999999-1.7882812499999998-0.534375-2.6765625-1.24453125l-1.14609375 1.24453125c1.14609375 0.94921875 2.42578125 1.4765625 3.8273437500000003 1.5960937499999999V18.0h0.9609375zm0.060937500000000006-5.146875c0.6375 0.17578125 1.021875 0.3515625 1.275 0.534375 0.18984374999999998 0.23906249999999998 0.31875 0.47109375000000003 0.31875 0.8296874999999999s-0.12890625 0.590625-0.44765625000000003 0.8296874999999999c-0.31875 0.23906249999999998-0.7054687500000001 0.3515625-1.14609375 0.41484374999999996V12.853124999999999zM10.528125 10.3125c-0.18984374999999998-0.17578125-0.31875-0.47109375000000003-0.31875-0.7664062500000001 0.0-0.3515625 0.12890625-0.590625 0.37968749999999996-0.8296874999999999 0.31875-0.23906249999999998 0.6375-0.3515625 1.14609375-0.41484374999999996v2.5453124999999996c-0.6375-0.1828125-1.0171875-0.35859375000000004-1.20703125-0.534375z" />
+</svg>
```

Comment:
Nice job adding an icon, but I would recommend sticking with the same icon family as the others for better visual consistency.

I used [Tabler](https://tablericons.com/) for the other icons; [here's](https://tablericons.com/icon/currency-dollar) an option for 'dollar' that would work:

<img width="595" alt="Screenshot 2025-04-06 at 19 51 44" src="https://github.com/user-attachments/assets/5e35104b-a254-487e-88fd-0b20e251e2b4" />



```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" width="24" height="24" stroke-width="2"> <path d="M16.7 8a3 3 0 0 0 -2.7 -2h-4a3 3 0 0 0 0 6h4a3 3 0 0 1 0 6h-4a3 3 0 0 1 -2.7 -2"></path> <path d="M12 3v3m0 12v3"></path> </svg> 
```